lib/get-bindings.d.ts

```
@@ -57,14 +57,74 @@ export declare const enum DomainGetXMLDescFlags {
+    NONE = 0,
+    ACPI = 1,
+    GUEST_AGENT = 2,
+    INIT = 4,
+    SIGNAL = 8,
+}
+
+export declare const enum DomainInterfaceAddressesSource {
+    LEASE = 0,
+    AGENT = 1,
+    ARP = 2,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)